docs/dxp/7.x/en/site-building/developer-guide/developing-page-fragments/creating-a-contributed-fragment-collection.md

```diff
@@ -9,14 +9,32 @@ Contributed Fragment Collections are deployable modules containing Page Fragment
 1. Add the Fragment Resources.
 1. Build the JAR and copy it to the Docker Container.
 
-This example uses a Docker image with a fresh install of Liferay DXP.
-
-> This example runs on Liferay DXP 7.3+
```

Comment:
I know you said no to blockquotes, but JR said to use that format til we have badges. So, I'm getting conflicting information from you two. I'm all for the change you made though.